.github/workflows/build-for-release.yml

```diff
@@ -109,6 +141,24 @@ jobs:
     env:
       INTERACTIVE: false
     steps:
+    - name: Get correct version number
+      run: | # bash
+        if [ '${{matrix.edition}}' == 'ee' ]; then
+          version='${{ needs.check-version.outputs.ee }}'
+        else
+          version='${{ needs.check-version.outputs.oss }}'
+        fi
+
+        echo "VERSION=$version" >> $GITHUB_ENV
+    - name: Setup Java ${{ needs.get-build-requirements.outputs.java_version }}
+      uses: actions/setup-java@v3
+      with:
+        distribution: 'temurin'
+        java-version: ${{ needs.get-build-requirements.outputs.java_version }}
+    - name: Setup Node.js ${{ needs.get-build-requirements.outputs.node_version }}
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ needs.get-build-requirements.outputs.node_version }}.x
```

Comment:
Is this trailing `.x` a typo or is it intentionally there?

Reply:
intentional. the docs use it 🤷 , I think it works without the `.x` but I thought it'd just follow the suggested pattern.